NeuroRuler/utils/img_helpers.py

```diff
@@ -167,6 +169,21 @@ def get_properties_from_sitk_image(img: sitk.Image) -> tuple:
     )
 
 
+def are_properties_eq(props1: tuple, props2: tuple) -> bool:
+    """:param props1:
+    :type props1: tuple
+    :param props2:
+    :type props2: tuple
+    :return: true if the two image properties are considered equal.
+    :rtype: bool"""
+    is_center_eq = props1[0] == props2[0]
+    is_size_eq = props1[1] == props2[1]
+    spacing1 = props1[2]
+    spacing2 = props2[2]
```

Comment:
I originally chose to use a tuple for properties, but indexing by number only is bad. We should instead use [named tuples](https://realpython.com/python-namedtuple/#creating-tuple-like-classes-with-namedtuple).

Reply:
I can do this later.